utils/render-rules.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+from glob import glob
+import collections
+import os
+import re
+import pathlib
+
+import argparse
+
+import ssg.build_yaml
+import ssg.controls
+import ssg.yaml
+import ssg.jinja
+import template_renderer
+
+
+class HtmlOutput(template_renderer.Renderer):
+    TEMPLATE_NAME = "rules-template.html"
+
+    def _get_all_compiled_profiles(self):
+        compiled_profiles = glob(str(self.built_content_path / "profiles" / "*.profile"))
+        profiles = []
+        for p in compiled_profiles:
+            profiles.append(ssg.build_yaml.Profile.from_yaml(p))
```

Comment:
could be a generator

Reply:
I don't understand this suggestion, please expand it.

Reply:
For example it can be a list comprehension but it's OK
 https://www.w3schools.com/python/python_lists_comprehension.asp

Reply:
I see, that's correct.